common/views/components/ArchiveTree/ArchiveTree.js

```diff
@@ -454,13 +459,22 @@ const ListItem = ({
   posInSet: number,
   tabbableId: ?string,
   setTabbableId: string => void,
+  setShowArchiveTree: boolean => void,
 |}) => {
   const { isKeyboard, isEnhanced } = useContext(AppContext);
   const isEndNode = item.children && item.children.length === 0;
   const isSelected =
     (tabbableId && tabbableId === item.work.id) ||
     (!tabbableId && currentWorkId === item.work.id);
   const toggles = useContext(TogglesContext);
+  const { updateEndRef } = useContext(ModalContext);
+  function updateTabbing(id) {
```

Comment:
Maybe a comment about what this function does? I get it to some degree, but probably only in context of this PR.